<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[We are in Android, now checking for Alert messages...]]>
    </line>
  </reporter-output>
  <suite name="POC_Automation_TestSuite" duration-ms="255971" started-at="2017-04-06T13:43:34Z" finished-at="2017-04-06T13:47:50Z">
    <groups>
      <group name="Testing">
        <method signature="LoginTests.onHoldWorkOrder()[pri:4, instance:mwo.tests.LoginTests@67b6d4ae]" name="onHoldWorkOrder" class="mwo.tests.LoginTests"/>
      </group> <!-- Testing -->
    </groups>
    <test name="Test" duration-ms="255971" started-at="2017-04-06T13:43:34Z" finished-at="2017-04-06T13:47:50Z">
      <class name="mwo.tests.LoginTests">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:mwo.tests.LoginTests@67b6d4ae]" name="beforeSuite" is-config="true" duration-ms="35916" started-at="2017-04-06T19:12:58Z" finished-at="2017-04-06T19:13:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:mwo.tests.LoginTests@67b6d4ae]" name="beforeClass" is-config="true" duration-ms="0" started-at="2017-04-06T19:13:34Z" finished-at="2017-04-06T19:13:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:mwo.tests.LoginTests@67b6d4ae]" name="beforeMethod" is-config="true" duration-ms="26175" started-at="2017-04-06T19:13:34Z" finished-at="2017-04-06T19:14:00Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void mwo.tests.LoginTests.onHoldWorkOrder() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="onHoldWorkOrder()[pri:4, instance:mwo.tests.LoginTests@67b6d4ae]" name="onHoldWorkOrder" duration-ms="221098" started-at="2017-04-06T19:14:00Z" finished-at="2017-04-06T19:17:42Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.appium.base.PageBase.enterTextinCommentsFeild(Unknown Source)
	at mwo.pages.OnHoldPage.enterCommentsInNotesField(Unknown Source)
	at mwo.pages.OnHoldPage.onHoldWO(Unknown Source)
	at mwo.tests.LoginTests.onHoldWorkOrder(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[We are in Android, now checking for Alert messages...]]>
            </line>
          </reporter-output>
        </test-method> <!-- onHoldWorkOrder -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:mwo.tests.LoginTests@67b6d4ae]" name="afterMethod" is-config="true" duration-ms="8620" started-at="2017-04-06T19:17:42Z" finished-at="2017-04-06T19:17:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=onHoldWorkOrder status=FAILURE method=LoginTests.onHoldWorkOrder()[pri:4, instance:mwo.tests.LoginTests@67b6d4ae] output=We are in Android, now checking for Alert messages...]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:mwo.tests.LoginTests@67b6d4ae]" name="afterClass" is-config="true" duration-ms="47" started-at="2017-04-06T19:17:50Z" finished-at="2017-04-06T19:17:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:mwo.tests.LoginTests@67b6d4ae]" name="afterSuite" is-config="true" duration-ms="454" started-at="2017-04-06T19:17:50Z" finished-at="2017-04-06T19:17:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- mwo.tests.LoginTests -->
    </test> <!-- Test -->
  </suite> <!-- POC_Automation_TestSuite -->
</testng-results>
